app/views/collections/show.html.erb

```diff
@@ -1,4 +1,6 @@
 <h1><%= @collection.name %></h1>
+</br>
```

Comment:
Do we need these line breaks any more?
It's probably a separate pull request to remove line breaks throughout the project, but you should probably stop introducing new ones.

Reply:
yeah - will defo stop doing it now I've started styling proper - I have a couple of styling stories in the backlog now... will make sure removing the /br is part of that work